app/javascript/listings/listings.jsx

```diff
@@ -284,27 +304,33 @@ export class Listings extends Component {
 
   listingSearch(query, tags, category, slug) {
     const t = this;
-    const { index, page, listings } = t.state;
-    const filterObject = { tagFilters: tags, hitsPerPage: 75, page };
+    const { page, listings } = t.state;
+    const dataHash = {
+      per_page: LISTING_PAGE_SIZE,
+      page,
+      classified_listing_search: query,
+    };
+
+    /* tags defaults to an empty String, so we check for that case here */
```

Comment:
I think we should use jsdoc to document this whole method, since we're in the code anyways.

Reply:
Updated!